templates/includes/sidebar.html

```diff
@@ -1,6 +1,21 @@
 <ul class="sidebar-nav">
+  {% if  user.is_authenticated %}
+    <li><a href="{% url 'taxi:driver-detail' user.pk %}">{{ user.username }}</a></li>
+  {% else %}
+    <li><a href="{% url "taxi:login" %}">Login</a></li>
+  {% endif %}
+  <br>
   <li><a href="{% url "taxi:index" %}">Home page</a></li>
   <li><a href="{% url "taxi:manufacturer-list" %}">Manufacturers</a></li>
   <li><a href="{% url "taxi:car-list" %}">Cars</a></li>
   <li><a href="{% url "taxi:driver-list" %}">Drivers</a></li>
+  <br>
+  {% if user.is_authenticated %}
+    <form action="{% url "taxi:logout" %}" method="post">
```

Comment:
Inconsistent use of quotation marks in the template tag. Consider using single quotes consistently for URL template tags, like `{% url 'taxi:logout' %}`.